c/cosmic-comp/pkg/0001-Allow-libdisplay-info-0.3.patch

```diff
@@ -1,38 +1,40 @@
-From 11397eb254fddd7ed5a88a02cb19da84c1cc46ff Mon Sep 17 00:00:00 2001
+From 2f002efc600186166ee1578ff8753a7495f5a6ea Mon Sep 17 00:00:00 2001
 From: Bryan Hyland <bryan.hyland32@gmail.com>
 Date: Wed, 17 Sep 2025 12:17:43 -0700
 Subject: [PATCH] Update libdisplay-info patch
 
```

Comment:
Either the patch or the recipe (where it references the patch) could benefit from an explanation along the lines of:

> Because our version of libdisplay-info is newer than what's supported by the upstream crate, we need to patch this a little. This is not upstreamable as is.